src/web/components/Onwards/ExactlyFive.tsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+
+import { Card } from '@frontend/web/components/Card/Card';
+import { UL } from '@frontend/web/components/Card/components/UL';
+import { LI } from '@frontend/web/components/Card/components/LI';
+
+type Props = {
+    content: TrailType[];
+};
+
+export const ExactlyFive = ({ content }: Props) => (
+    <>
+        <UL direction="row" bottomMargin={true}>
+            <LI padSides={true} percentage="33%">
+                <Card
+                    {...{
+                        linkTo: content[0].url,
```

Comment:
I assume `content[0]` is typesafe?

Reply:
Yes, should be. Typescript knows what properties should be available on that array object.